doc/releases/changelog-0.11.0.md

```diff
@@ -2,10 +2,13 @@
 
 <h3>New features since last release</h3>
```

Comment:
maybe I am missing it somewhere, but qubit-invariant compilation should probably be in the new features section

Reply:
Agreed!